ui/imports/shared/views/PasswordView.qml

```diff
@@ -16,12 +16,13 @@ Column {
 
     property bool ready: newPswInput.text.length >= root.minPswLen && newPswInput.text === confirmPswInput.text && errorTxt.text === ""
     property int minPswLen: 10
+    property int showPswLen: 2
     property bool createNewPsw: true
     property string title: qsTr("Create a password")
     property bool titleVisible: true
     property string introText: qsTr("Create a password to unlock Status on this device & sign transactions.")
     property string recoverText: qsTr("You will not be able to recover this password if it is lost.")
-    property string strengthenText: qsTr("Minimum 6 characters. To strengthen your password consider including:")
+    property string strengthenText: qsTr("Minimum 10 characters. To strengthen your password consider including:")
```

Comment:
I would suggest to use the `root.minPswLen` param instead of the explicit number.